src/scenes/home/landing/triGroup/triGroup.css

```diff
@@ -1,9 +1,24 @@
-.videos {
-  display: flex;
-  justify-content: space-around;
-  align-items: center;
-  border-radius: 4px;
+.embedContainer {
+  width: 100%;
+  max-width: 960px;
+}
+
+.embedVideo {
+  position: relative;
+  padding-bottom: 56.25%;
```

Comment:
Nice usage of the `56.25%` rule to preserve the aspect ratio!

Reply:
Can you tell me more about this out of curiosity?

Reply:
From the pioneer https://alistapart.com/article/creating-intrinsic-ratios-for-video